modules/model/src/main/scala/io/janstenpickle/trace4cats/model/SemanticAttributeKeys.scala

```diff
@@ -0,0 +1,25 @@
+package io.janstenpickle.trace4cats.model
+
+object SemanticAttributeKeys {
+  // local service attributes
+  private final val service = "service."
+  final val servicePort: String = service + "port"
+  final val serviceIpv4: String = service + "ipv4"
+  final val serviceIpv6: String = service + "ipv6"
+  final val serviceHostname: String = service + "hostname"
```

Comment:
I purposefully didn't prepend these with `local` as I think we should be biased towards the local system